### ISSUE TITLE
Standardised test. Benchmarking AOflagger versions <2.8 & 2.9+ regimes

### ISSUE THREAD

**Issue by jradcliffe5** [e-merlin/eMERLIN_CASA_pipeline#28]
I'll make some plots of time vs. memory with the 2.7 & 2.9 versions (i.e. forced mms shape compared to optimised). I guess I can do the same for no. of cores with different available RAM vs time.

I guess in the end we would want to test this on a 24hr run i.e. a ~300GB data set. I don't have that time on this computer atm but I will do the tests on a 106GB data set I have here.

**Comment by jradcliffe5** [comment]
I started with the use of aoflagger fields with version 2.9. The tar file contains the plots of usage & cpu that Javier wrote. I was refraining from doing the rest of the tests with ms until issue #36 is resolved. 

There are three folders with have runs with a 180GB L-band test data set. To save time, I only ran the hanning smoothing, aoflagger_fields and ms2mms. Each folder contains runs using 2, 4, 8, 32, and 64 cores and the folder name is the amount of memory available to each run. The killer is of course when aoflagger cannot directly read the files into memory!

The tar file can be found http://www.jb.man.ac.uk/~radcliff/exchange/CASA_pipeline/ (the hyperlink doesn't work because of the stupid tilda~...)
